formats/config/section.go

```diff
@@ -0,0 +1,76 @@
+package config
+
+import "strings"
+
+type Section struct {
+	Name        string
+	Options     Options
+	Subsections Subsections
+}
+
+type Subsection struct {
+	Name    string
+	Options Options
+}
+
+type Sections []*Section
```

Comment:
There might be multiple sections with the same name. It's up to us to offer convenience methods in the future that hide this for common use case, but the structure needs to preserve this.